feature-group-definitions/anchor-positioning.dist.yml

```diff
@@ -0,0 +1,8 @@
+# Generated from: anchor-positioning.yml
```

Comment:
This file demonstrates dist-ing a feature where there's no status, `compat_features`, or tags in BCD.

Reply:
It's nice to see this working, but can you drop it from this PR? It would be nice to treat the absence of a `.dist.yml` as a burndown list for statuses being available.

Reply:
Done with 54a8a10.